var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -56,7 +56,7 @@ class Visit(CMakePackage):
     ############################
     homepage = "https://wci.llnl.gov/simulation/computer-codes/visit/"
     git      = "https://github.com/visit-dav/visit.git"
-    url = "https://github.com/visit-dav/visit/releases/download/v3.2.1/visit3.2.1.tar.gz"
+    url = "https://github.com/visit-dav/visit/releases/download/v3.2.2/visit3.2.2.tar.gz"
```

Comment:
FYI. This change isn't necessary since Spack will substitute the version automatically.